### PR TITLE
Add UI for accelerometer calibration

### PR DESCRIPTION
**Pull request by Williangalvani**
depends on #2463

### CODE REVIEW

**Review by patrickelectric — APPROVED** [review]

**Review by joaoantoniocardoso — COMMENTED** [review]
Lol, Patrick merged before I finished the review, but:

1. Some small UI problems:
  - The dialogue titles are hard to read in dark mode (and just ugly anyway)
  - (Maybe) the fonts are smaller than they should be for a dialogue?
2. Quick Calibration works fine.
3. Full calibration:
  - It doesn't show a success message
  - It doesn't let me close the dialogue

**Comment by joaoantoniocardoso** [comment]
Another one:
- The `next` button in the full calibration dialogue is clickable multiple times.

File: core/frontend/src/components/vehiclesetup/configuration/accelerometer/FullAccelerometerCalibration.vue [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    },
+    show_start_button() {
+      return [CalState.IDLE, CalState.WAITING_FOR_VEHICLE_RESPONSE].includes(this.state)
+    },
+    start_button_loading() {
+      return this.state === CalState.WAITING_FOR_VEHICLE_RESPONSE
+    },
+    current_state_text() {
+      const AccelStateText: { [key: number]: string } = {
+        [CalState.WAITING_FOR_LEVEL_POSITION]: 'Place the vehicle on a level surface',
```
Comment (joaoantoniocardoso) [inline core/frontend/src/components/vehiclesetup/configuration/accelerometer/FullAccelerometerCalibration.vue:118]: The message isn't clear enough: we _must_ say what position we expect the vehicle to be. A _level_ surface can mean anything.

File: core/frontend/src/components/vehiclesetup/configuration/accelerometer/QuickAccelerometerCalibration.vue [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+    <v-card>
+      <v-card-title class="text-h5 grey lighten-2">
+        Accelerometer Calibration
+      </v-card-title>
+      <v-card-text>
+        <div>
+          Quick calibration requires that you place the vehicle on a level surface and keep it still for a few seconds.
+          It is usually enough if your vehicle is generally leveled while in use.
+        </div>
```
Comment (joaoantoniocardoso) [inline core/frontend/src/components/vehiclesetup/configuration/accelerometer/QuickAccelerometerCalibration.vue:25]: why do we need a div here?

**Pull request merged**